impeller/geometry/path.h

```diff
@@ -200,10 +199,13 @@ class Path {
   // builder from affecting the existing taken paths.
   struct Data {
     Data() = default;
-    Data(const Data& other) = default;
+
+    Data(Data&& other) = default;
 
     ~Data() = default;
 
+    Data Clone() const;
```

Comment:
Document this - what's getting copied, is it copy on write, etc.

Reply:
Everything - Done